meteor/client/ui/SegmentTimeline/SegmentTimelineContainer.tsx

```diff
@@ -615,6 +622,21 @@ export const SegmentTimelineContainer = translateWithTracker<IProps, IState, ITr
 			window.removeEventListener('resize', this.onWindowResize)
 		}
 
+		private getSegmentBudgetDuration(): number | undefined {
```

Comment:
I'm somewhat confused. So `RundownTimingCalculator` seems to already be doing almost the same thing, and yet then the Segment component seems to be doing it again? And then it feels like it only does that to pass that information forward to components that have access to the timing context anyway, so they could just get it from the Calculator. Or am I missing something?